pkg/mimirtool/commands/remote_read.go

```diff
@@ -205,7 +211,7 @@ func (c *RemoteReadCommand) readClient() (remote.ReadClient, error) {
 		Timeout: model.Duration(c.readTimeout),
 		HTTPClientConfig: config_util.HTTPClientConfig{
 			BasicAuth: &config_util.BasicAuth{
-				Username: c.tenantID,
+				Username: c.apiUser,
```

Comment:
I do understand how confusing this could be, but I wonder if these changes are 100% expected for a user. e.g. I use tenant-id to authenticate myself in Mimir's GEM instance (ref [docs](https://grafana.com/docs/enterprise-metrics/latest/set-up/set-up-gem-tenant/)).

Would it be even more confusing if we did `cmp.Or(user, id)` here?

Reply:
the cmp.or look to resole @dimitarvdimitrov  remark to keep same behaviour as before.